recipes/dbus/1.x.x/conanfile.py

```diff
@@ -92,8 +97,8 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
 
-        for i in ["var", "share", "bin", "etc"]:
-            tools.rmdir(os.path.join(self.package_folder, i))
+        for i in ["var", "share", "etc"]:
+            shutil.move(os.path.join(self.package_folder, i), os.path.join(self.package_folder, "res", i))
```

Comment:
Can this move be avoided by doing `self.cpp_info.resdires = ["var", "share", "etc"]` in `package_info`?

Reply:
Not until `share` is allowed in general https://github.com/conan-io/conan-center-index/issues/3288

Reply:
Dammit!

@uilianries
How about allowing `share` when it is in some *allowlist* variable?